package/MDAnalysis/analysis/encore/similarity.py

```diff
@@ -858,14 +848,22 @@ def hes(ensembles,
          [ 7032.19607004     0.        ]]
     """
 
+    if weights == 'mass':
+        weights = ['mass' for _ in range(len(ensembles))]
+    elif weights is not None:
+        if len(weights) != len(ensembles):
+            raise ValueError("need weights for every ensemble")
+    else:
+        weights = [None for _ in range(len(ensembles))]
```

Comment:
@mtiberti, @wouterboomsma is this correct to demand for `hes` that custom weights have to be specified for each ensemble?